styles/main/_content.scss

```diff
@@ -297,8 +297,8 @@
 
 		.album,
 		.photo {
-			// 3 thumbnails per row
-			--size: calc((100vw - 3px) / 3);
+			// 2 thumbnails per row
+			--size: calc((100vw - 5px) / 2);
```

Comment:
That's something we need to discuss and I have to test on various smartphones, resp. For example, I am very happy with 3 photos per row on my phone and 2 photos only would be a waste of space.

Maybe we need to be more specific about the the intended screen width, at the moment  3 thumbnails per row is used for devices with a screen width between 320px and 567px. Maybe we have to use more cases here, i.e. use 2 thumbnails per row for 320px...440px and 3 thumbnails for 440px...567px. (Note, I just invented the 440px for the example, it is not an informed choice.)